src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int ballsToCreate = 3;
```

Comment:
Consider defining `ballsToCreate` as a constant, as it is a magic number. This will improve code readability and maintainability. For example, you can define it as `private static final int BALLS_TO_CREATE = 3;`.